src/com/opc_ua/opcua_objectstruct_helper.cpp

```diff
@@ -144,6 +152,8 @@ bool COPC_UA_ObjectStruct_Helper::addOPCUAStructTypeComponent(UA_Server *paServe
     UA_QUALIFIEDNAME(mOpcuaTypeNamespaceIndex, &structMemberName[0]),
     UA_NODEID_NUMERIC(0, UA_NS0ID_BASEDATAVARIABLETYPE), vAttr, nullptr, &memberNodeId);
 
+  UA_String_clear(&memberBrowsePath);
+  UA_NodeId_clear(&memberNodeId);
```

Comment:
Is this the correct order for clearing these allocations? Without having looked into the documentation this looks like it's the wrong way around?